spec/semantics.html

```diff
@@ -550,7 +550,7 @@
         1. Let _ye_ be _y_'s _endIndex_.
         1. <ins>If _direction_ is equal +1, then</ins>
           1. Let _s_ be a fresh List whose characters are the characters of _Input_ at indices _xe_ (inclusive) through _ye_ (exclusive).
-        1. <ins>Else, _direction_ is equal to -1.</ins>
+        1. <ins>Else, let _direction_ is equal to -1.</ins>
```

Comment:
I don't understand this insertion; I guess we could write "Assert" here.

Reply:
Yeah, I misunderstood.

Reply:
I like the last form in @claudepache’s comment — that makes this very explicit (and would’ve prevented my confusion).